crypto/evp/p_rsa_asn1.c

```diff
@@ -108,6 +108,19 @@ static int rsa_pub_decode(EVP_PKEY *out, CBS *params, CBS *key) {
   return 1;
 }
 
+static int rsa_pss_pub_decode(EVP_PKEY *out, CBS *params, CBS *key) {
+  // TODO(shang): add pss paramters decode.
+  RSA *rsa = RSA_parse_public_key(key);
+  if (rsa == NULL || CBS_len(key) != 0) {
```

Comment:
Prefer encapsulating conditions.

Reply:
Can a sample code added? I don't get the exact concern here. It's also existing code.

Reply:
Replied above. This actually hurts readability.